scripts/wdl/README.md

```diff
@@ -6,5 +6,70 @@ https://github.com/broadinstitute/wdl
 
 https://github.com/broadinstitute/cromwell
 
-To get started, copy the template.json for the workflow you wish to run and adjust parameters accordingly.
+#### Setting up parameter json file for a run
+
+To get started, copy the relevant ``*.template.json`` for the workflow you wish to run and adjust parameters accordingly.  This file has reasonable default parameters for exome or targeted panels.  
+- For WGS, please see notes below for ``isWGS``, ``wgsBinSize``, ``seg_param_undoSplits``, and ``seg_param_undoSD``
+- Values starting with ``$__`` *must* be replaced with values for your run.  Please note that python [templates](https://docs.python.org/2/library/string.html#template-strings) can be useful for replacing these values
+
+
+#### Explanation of fields in case workflow
+
+Fields marked as Advanced almost never require a user to adjust the default value.
+Reference used must be the same between PoN and case samples
+
+- ``case_gatk_acnv_workflow.input_bam_list`` -- tsv file consisting of corresponding bam files and entity names as described in case_gatk_acnv_workflow.wdl
+- ``case_gatk_acnv_workflow.CalculateTargetCoverage.grouping`` -- (Advanced) Always choose "SAMPLE",
+- ``case_gatk_acnv_workflow.common_snp_list`` --  SNP list to use in het pulldown 
+- ``case_gatk_acnv_workflow.ref_fasta_fai`` --  Path to reference fasta fai file.  Broad internal: /seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta.fai 
+- ``case_gatk_acnv_workflow.target_file`` --  same target file (NOT bed format) as was used for the PoN creation
+- ``case_gatk_acnv_workflow.gatk_jar`` --  absolute path to gatk-protected.jar 
+- ``case_gatk_acnv_workflow.PadTargets.mem`` -- (Advanced) Number of GB RAM to use when running PadTargets.  Always use 1, unless your target file is really large
```

Comment:
Nitpicking, but I would be consistent about whether or not to end all bullet points with a period.